app/TracingStrategyContext.spec.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useContext } from 'react';
+import { View, Text } from 'react-native';
+import { render, cleanup } from '@testing-library/react-native';
+import '@testing-library/jest-native/extend-expect';
```

Comment:
introduced jest-native as it makes working with react-testing-library __a lot__ easier.